packages/cashier/src/Components/Error/__tests__/transfer-locked.spec.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Router } from 'react-router';
+import { createBrowserHistory } from 'history';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { Checklist } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import TransferLocked from '../transfer-locked';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+describe('TransferLocked', () => {
+    it('Should show the default lock content if the account is not financial', () => {
+        render(
+            <TransferLocked
+                is_financial_account={false}
+                is_financial_information_incomplete
+                is_trading_experience_incomplete
+            />
+        );
+        expect(screen.getByText('Transfers are locked')).toBeInTheDocument();
+    });
+
+    it('Should show the default lock content if the account is financial and the account information is incomplete', () => {
+        render(
+            <TransferLocked
+                is_financial_account
+                is_financial_information_incomplete={false}
```

Comment:
According your test case description "is_financial_information_incomplete" property should be "true", i think that we should change test case description to "... and the account information is complete", instead of "incomplete".

Reply:
@george-usynin-deriv What I meant was to have incomplete account information which contains both `financial_information` and `trading_experience` flags and not only `financial_information`.
Since the description wasn't clear and made confusion I changed it, please take a look again🙏

Reply:
@Bahar-FS LGTM 👍